doc/tutorials/setup_logical_partition_scenario_on_kbl_nuc_i7.rst

```diff
@@ -0,0 +1,154 @@
+.. _setup_logical_partition_scenario_on_kbl_nuc_i7:
+
+Guidelines of Setup Logical Partition Scenario on KBL NUC i7
+############################################################
+
+ACRN hypervisor supports logic partition scenario, in which the User OS (such as Clear Linux) running in a pre-launched VM can bypass the ACRN hypervisor and directly access isolated PCI devices. The following guidelines provide step by step instructions on how to set up the ACRN hypervisor logical partition scenario on KBL NUC i7 boards running two pre-launched VMs. The logical partition scenario on KBL NUC i7 is shown in :numref:`logical_partition_scenario_on_kbl_nuc_i7`.
+
+.. figure:: images/logical_partition_scenario_on_kbl_nuc_i7.png
+   :align: center
+   :width: 400px
+   :name: logical_partition_scenario_on_kbl_nuc_i7
+
+   The logical partition scenario on KBL NUC i7
+
+Prerequisites
+*************
+
+Disable "Intel Hyper Threading Technology" in the BIOS.
+
+In the logical partition scenario, two pre-launched VMs (running Clear Linux) are started by the ACRN hypervisor. To set up the Linux root filesystems for each VM, follow the Clear Linux OS bare metal installation guide to install Clear Linux OS on a SATA disk and a USB flash disk prior the setup, as the two pre-launched VMs will mount the root file systems via the SATA controller and the USB controller respectively.
```

Comment:
@leilu please use "a storage device with USB interface  (such as USB Flash or SATA disk connecting with USB3.0 SATA converter etc)" instead of "a USB flash", and check it in this whole document.

Reply:
Could someone use a SATA and NVMe drive instead for example? I believe the underlying requirement is to use controllers that are distinct PCI devices, is that correct?

Reply:
@gvancuts Yes, the requirement is to use SATA controller and USB controller.

Reply:
fix at doc v2 version

Reply:
fix at doc v2 version.